### PR TITLE
ticdc: Update cdc server config

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14992
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-server-config.md [hunk 1]
```diff
@@ -45,30 +46,37 @@ cluster-id = "default"
...
 # [kv-client]
-#  worker-concurrent = 8
-#  worker-pool-size = 0
-#  region-scan-limit = 40
-#  region-retry-duration = 60000000000
+#   The number of threads that can be used in a single Region worker. This parameter is optional and its default value is 8.
+#   worker-concurrent = 8
+#   The number of threads in the shared thread pool of TiCDC, mainly used for processing KV events. This parameter is optional and its default value is 0, indicating that the default pool size is twice the number of CPU cores.
+#   worker-pool-size = 0
+#   The retry time of Region connection. This parameter is optional and its default value is `60000000000` nanoseconds (that is, 1 minute). You can configure this parameter in two ways: specifying only the number (for example, configuring it as `50000000` represents 50000000 nanoseconds, which is 50 milliseconds), or specifying both the number and unit (for example, directly configuring it as `50ms`).
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:80]:
```suggestion
#   The retry duration of Region connection. This parameter is optional and its default value is `60000000000` nanoseconds (that is, 1 minute). You can configure this parameter in two ways: specifying only the number (for example, configuring it as `50000000` represents 50000000 nanoseconds, which is 50 milliseconds), or specifying both the number and unit (for example, directly configuring it as `50ms`).
```

**Review by sdojjy — APPROVED** [review]

File: ticdc/ticdc-server-config.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ The following are descriptions of options available in a `cdc server` command:
 The following are configurations in the configuration file of `cdc server`:
```
Comment (Oreoxmt) [inline ticdc/ticdc-server-config.md:31]:
```suggestion
The following describes the configuration file specified by the `config` option in the `cdc server` command:
```

File: ticdc/ticdc-server-config.md [hunk 3]
```diff
@@ -31,6 +31,7 @@ The following are descriptions of options available in a `cdc server` command:
...
 
 ```yaml
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:33]:
```suggestion
```toml
```

File: ticdc/ticdc-server-config.md [hunk 4]
```diff
@@ -45,30 +46,37 @@ cluster-id = "default"
...
+#[log]
+# The output location for internal error logs of the zap log module. This parameter is optional and its default value is "stderr".
+# error-output = "stderr"
+# [log.file]
+#   The maximum size of a single log file, measured in MiB. This parameter is optional and its default value is 300.
+#   max-size = 300 # 300 MiB
+#   The maximum number of days to retain log files. This parameter is optional and its default value is `0`, indicating never to delete.
+#   max-days = 0
+#   The number of log files to retain. This parameter is optional and its default value is `0`, indicating to keep all log files.
+#   max-backups = 0
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:67]:
```suggestion
# [log]
# # The output location for internal error logs of the zap log module. This parameter is optional and its default value is "stderr".
#   error-output = "stderr"
#   [log.file]
#     # The maximum size of a single log file, measured in MiB. This parameter is optional and its default value is 300.
#     max-size = 300 # 300 MiB
#     # The maximum number of days to retain log files. This parameter is optional and its default value is `0`, indicating never to delete.
#     max-days = 0
#     # The number of log files to retain. This parameter is optional and its default value is `0`, indicating to keep all log files.
#     max-backups = 0
```

File: ticdc/ticdc-server-config.md [hunk 5]
```diff
@@ -45,30 +46,37 @@ cluster-id = "default"
...
-  num-concurrent-worker = 4
-  chunk-size-limit = 999
-  max-memory-percentage = 30
-  max-memory-consumption = 17179869184
-  num-workerpool-goroutine = 16
-  sort-dir = "/tmp/sorter"
+#[sorter]
+# The size of the shared pebble block cache in the Sorter module for the 8 pebble DBs started by default, measured in MiB. The default value is 128.
+# cache-size-in-mb = 128
+# The directory where sorter files are stored relative to the data directory. This parameter is optional and its default value is "/tmp/sorter".
```
Comment (Oreoxmt) [inline ticdc/ticdc-server-config.md:72]:
```suggestion
# The directory where sorter files are stored relative to the data directory (`data-dir`). This parameter is optional and its default value is "/tmp/sorter".
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**